CHANGELOG.md

```diff
@@ -239,6 +241,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Specific ERC-20 Portal
 - Deprecated mock contracts
 
+[1.0.1]: https://github.com/cartesi/rollups/releases/tag/v1.0.1
```

Comment:
[Keep a changelog](https://keepachangelog.com/en/1.0.0/) suggests adding a link for  `Unreleased` as well.
See https://github.com/cartesi/rollups-contracts/pull/39.